### PR TITLE
TIDAL-460 timeTracking.list sorting

### PR DESCRIPTION
**Pull request by Jdww**
Removed sort parameter which was not used and added a description explaining the sorting on timeTracking.list

### CODE REVIEW

**Review by jnstr — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -4177,7 +4177,7 @@ Time tracking allows users to record time spent on tasks or projects and the typ
 
 ### timeTracking.list [GET /timeTracking.list]
 
-Get a list of tracked time.
+Get a list of tracked time. Per day, time tracking entries with an automatically generated start time are displayed first (duration time tracking), followed by entries with a user generated start and stop time (start-stop time tracking), sorted ascending by start time.
```
Comment (jnstr) [inline apiary.apib:4180]: > with an automatically generated start time

Is this correct though? Duration time tracking has no start time, right?

Comment (adognameddog) [reply apiary.apib:4180]: It does. The api always returns a start time and a stop time.

**Review by adognameddog — APPROVED** [review]
:hourglass: Manually checking...

**Comment by adognameddog** [comment]
:white_check_mark: &nbsp;I am a dog named Dog and I approve this message

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**